pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -27,13 +27,14 @@ import (
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 
+	"path"
```

Comment:
This looks strange to me.
For me imports are splitted into three parts:
1. go standard lib, like "fmt", "sync".
2. third party lib, like "github.com/golang/glog"
3. k8s packages

Why "path" import is different from previous "fmt", "math", "sync" ?

Reply:
Yeah,I think you found a bug of goimports, however, when I manually modified this, it didn't use any hints.